.slim-lint.yml

```diff
@@ -0,0 +1,6 @@
+exclude:
+  - "**/lib/templates/slim/**/*"
+
+linters:
+  LineLength:
+    enabled: false
```

Comment:
Slim views, by their nature, seem to have extremely long lines. I have not found a way around this that keeps the syntax easy to read. For example, how could this be made shorter without sacrificing readability?

```slim
    meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
```

Since I haven't found a good solution, I opted to disable the `LineLength` linter entirely.